### ISSUE TITLE
Connection failed: transport: authentication handshake failed: x509: certificate has expired or is not yet valid

### ISSUE THREAD

**Issue by yujinkim-msft** [Azure/iotedge-eflow#225]
Hello Team, 

I'm using EFLOW 1.4LTS and facing the following issue. The service was running for some months, but currently when the we try to run the command Connect-EflowVm or any other command to interact with the VM I get the following: 

PS C:\\Users\\A001717 Connect-EflowVm
 - Configuration 'clientCertificateThumbprint' is not present. A default will be applied.
 - Configuration 'serverCertificateThumbprint' is not present. A default will be applied.
 - Configuration 'serverCertificateThumbprint' is not present. A default will be applied.
 - Configuration 'clientCertificateThumbprint' is not present. A default will be applied. 
 
[01/03/2023 11:24:08] Exception caught!!!  

- F:\\Azure IoT Edge\\nodectl.exe  compute vm list -o tsv --query '[*].name' failed to execute [Error: rpc error: code = Unavailable desc = connection error: desc = 'transport: authentication handshake failed: x509: certificate has expired or is not yet valid: current time 2023-01-03T11:24:08+01:00 is after 2022-11-20T09:51:03Z'] at line 3939 

[01/03/2023 11:24:08] Exception caught!!!  - Failed to SSH into virtual machine. Virtual machine name could not be retrieved. at line 2789

**Comment by fcabrera23** [comment]
Hi @yujinkim-msft,

Thanks for reaching out. On some of our older deployments, we are noticing a communication handshake failure due to the expiry of a previously installed cert. Please check [1.4.3.23013 Release notes](https://github.com/Azure/iotedge-eflow/releases/tag/1.4.3.23013) for the workaround.

Thanks,
Francisco

**Comment by chokosticks** [comment]
Hi @fcabrera23,

I ran into the same issue on version 1.4.1.13112 trying to run Connect-EflowVM. We've this on multiple installations.

![image](https://github.com/Azure/iotedge-eflow/assets/8739607/19f6ab28-f9a3-4266-95ab-fdf603b8c162)

But didn't succeed running updateWssdCert.ps1.

![image](https://github.com/Azure/iotedge-eflow/assets/8739607/f27a0f11-ee81-41ce-8697-e88b03e3453e)

I did try to add -Force -Confirm flags so it succeeds to stop the service.
Though it seems not to find the path \.wssd\nodectl.

On this machine the .wssd folder does exist, but it's empty.


Any thoughts how to go forward?
